src/renderer/pages/ConnectForm.tsx

```diff
@@ -167,6 +210,48 @@ const ConnectForm: FC<Props> = () => {
     }
   };
 
+  const saveCertText = (value: string): void => {
+    setCertText(value);
+    setConnection((oldConnection) => {
+      set(oldConnection, 'clientCert.cert', new TextEncoder().encode(value));
+      return oldConnection;
+    });
+  };
+
+  const saveKeyText = (value: string): void => {
+    setKeyText(value);
+    setConnection((oldConnection) => {
+      set(oldConnection, 'clientCert.key', new TextEncoder().encode(value));
```

Comment:
why do we do it via `lodash` and not via standard setters provided?

Reply:
clientCert could be undefined because of the nested protobuf and lodash protects against that

Reply:
also there are no standard setters with protos-ts it is just json.